CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 
+- **[BREAKING CHANGE]** Remove `size` from `SuccessModal` api
```

Comment:
I would split the breaking change bit in a separate and as-small-as-possible changeset.